templates/base.j2

```diff
@@ -14,7 +14,7 @@
 </head>
 <body>
 
-  {{ iconsSvg }}
```

Comment:
So, if this was working in an older version of Nunjucks, I expect it would have exactly the same security implications—it would have worked by implicitly treating this data as safe. So, if it hasn't been an issue, maybe it's not a problem? This only exposes things that are under the control of the person building the docs, yeah?